tests/scripts/all.sh

```diff
@@ -886,6 +888,63 @@ clang_version() {
     fi
 }
 
+gcc_version() {
+    cc="$1"
+    if command -v clang > /dev/null ; then
+        "$cc" --version | sed -En '1s/^[^ ]* \([^)]*\) ([0-9]+).*/\1/p'
+    else
+        echo 0  # report version 0 for "no clang"
```

Comment:
Huh? Is this `gcc_version` or `clang_version` or `generic_compiler_version`?

Reply:
Oof, sorry about the messy copypasta.
It's supposed to be gcc specific - will fix it.